data/json/effects.json

```diff
@@ -681,6 +681,7 @@
         "apply_memorial_log": "Contracted a fungal infection.",
         "remove_memorial_log": "Cured the fungal infection.",
         "miss_messages": [["You feel sick inside.", 1]],
+        "speed_name": "Fungal infection",
```

Comment:
Some of the speed_names are actually lowercase, for instance "Spore covered". I'm of the opinion that only the first word should be capitalized, but if there's a proper way to do it, I'll change to that.